frontend/src/components/FileUploader.js

```diff
@@ -8,68 +8,126 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDropzone } from 'react-dropzone';
-
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTimes, faCircle } from '@fortawesome/free-solid-svg-icons';
-import { Tag, Button, Grid } from '@trussworks/react-uswds';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@trussworks/react-uswds';
+import uploadFile from '../fetchers/File';
 
 import './FileUploader.css';
 
 function Dropzone(props) {
-  const { onChange } = props;
+  const { onChange, id, reportId } = props;
   const onDrop = (e) => {
+    let attachmentType;
+    if (id === 'attachments') {
+      attachmentType = 'ATTACHMENT';
+    } else if (id === 'other-resources') {
+      attachmentType = 'RESOURCE';
+    }
+    e.forEach(async (file) => {
+      try {
+        const data = new FormData();
+        data.append('reportId', reportId);
+        data.append('attachmentType', attachmentType);
+        data.append('file', file);
+        await uploadFile(data);
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.log(error);
+      }
+    });
     onChange(e);
   };
   const { getRootProps, getInputProps } = useDropzone({ onDrop });
 
-  // I tried moving these styles to a css file and applying a class to the container
-  // and span. The styles were not being applied, it seems like the Dropzone library
-  // is messing with the styles somewhere
-  const containerStyle = {
-    maxWidth: '21rem',
-    height: '8rem',
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
-    borderStyle: 'dashed',
-    borderWidth: '0.125rem',
-    borderColor: '#979797',
-  };
-
-  const textStyle = {
-    textAlign: 'center',
-    fontSize: '16px',
-  };
-
-  const linkStyle = {
-    cursor: 'pointer',
-    color: 'blue',
-    textDecoration: 'underline',
-  };
-
   return (
     <div
       {...getRootProps()}
-      style={containerStyle}
     >
       <input {...getInputProps()} />
-      <p style={textStyle}>
-        <b>Drag and drop your files here</b>
-        {' '}
-        <br />
-        or
-        <br />
-        <span style={linkStyle}>Browse files</span>
-      </p>
+      <button type="button" className="usa-button">
+        Browse files
+      </button>
     </div>
   );
 }
 
 Dropzone.propTypes = {
   onChange: PropTypes.func.isRequired,
+  reportId: PropTypes.node.isRequired,
+  id: PropTypes.string.isRequired,
 };
 
-const FileUploader = ({ onChange, files }) => {
+const FileTable = ({ onFileRemoved, files }) => {
+  let msg;
+  if (files.length === 0) {
```

Comment:
You can accomplish this by checking `files.length === 0` directly in the jsx (an [example](https://github.com/adhocteam/Head-Start-TTADP/blob/main/frontend/src/components/Navigator/index.js#L126))

Reply:
cool